sdk/eventhub/azure-eventhub/tests/livetest/synctests/test_send.py

```diff
@@ -179,3 +179,59 @@ def test_send_with_create_event_batch_with_app_prop_sync(connstr_receivers):
             received.extend(r.receive_message_batch(timeout=5000))
         assert len(received) >= 1
         assert EventData._from_message(received[0]).properties[b"raw_prop"] == b"raw_value"
+
+
+@pytest.mark.liveTest
+def test_send_list(connstr_receivers):
+    connection_str, receivers = connstr_receivers
+    client = EventHubProducerClient.from_connection_string(connection_str)
+    payload = "A1"
+    with client:
+        client.send_batch([EventData(payload)])
+    received = []
+    for r in receivers:
+        received.extend([EventData._from_message(x) for x in r.receive_message_batch(timeout=10000)])
+
+    assert len(received) == 1
+    assert received[0].body_as_str() == payload
+
+
+@pytest.mark.liveTest
+def test_send_list_partition(connstr_receivers):
+    connection_str, receivers = connstr_receivers
+    client = EventHubProducerClient.from_connection_string(connection_str)
+    payload = "A1"
+    with client:
+        client.send_batch([EventData(payload)], partition_id="0")
+        message = receivers[0].receive_message_batch(timeout=10000)[0]
+        received = EventData._from_message(message)
+    assert received.body_as_str() == payload
+
+
+@pytest.mark.liveTest
+def test_send_list_too_large(connection_str):
```

Comment:
Potentially naive question as I think through test cases:  I've called out in SB how an empty list of events to a batch results in a slightly opaque error, does that get communicated nicely here?  Is it worth validating?  (If it falls into one of the existing ValueError scenarios we're probably OK, but like with my comment in SB, probably worth making sure that the exception message is clear.)

Reply:
Adding empty EventDataBatch check in EH will cause a break change. An empty list / EventDataBatch will raise an EventDataSendError. Users won't send an empty list/batch unless they play to do so. So I think we don't need to add this check now in EH.